build-tools/packages/build-cli/src/commands/merge/branches.ts

```diff
@@ -446,11 +446,13 @@ The aim of this pull request is to sync ${source} and ${target} branch. This bra
 1. Acknowledge the pull request by adding a comment -- "Actively working on it".
 2. Merge ${mergeBranch} into the target branch, ${target}. **The direction of the merge matters!** You need to checkout ${target} and merge ${mergeBranch} into it, then fast-forward ${mergeBranch} to the merge commit. To do that use the following git commands:
   - \`git fetch --all\` -- this ensures your remote refs are updated
+  - \`git remote -v\` -- displays the list of remote repositories associated with your Git repository along with their corresponding URLs. You have to choose the remote repository associated to URL: **https://github.com/microsoft/FluidFramework.git**. **Please repalce your remote accordingly.**
+  Assuming ${remote} is the remote associated to **https://github.com/microsoft/FluidFramework.git**
```

Comment:
where is the value of remote defined. i wonder if you should just tell people to create a new remote for merging, and give it a unique name, and instructions to create, then people who don't want to think about it, or have a very unique setup could create that remote for merging. advanced users could substitute their own remote name if they care to.

Reply:
something like `git add remote mainNextMerge https://github.com/microsoft/FluidFramework.git`

Reply:
Remote is passed as a flag: https://github.com/microsoft/FluidFramework/blob/main/build-tools/packages/build-cli/src/commands/merge/branches.ts#L50 and defined here: https://github.com/microsoft/FluidFramework/blob/main/build-tools/packages/build-cli/src/commands/merge/branches.ts#L114

Reply:
added this git command:
`git remote add mainNextMerge https://github.com/microsoft/FluidFramework.git`

Reply:
if `mainNextMerge` remote already exists then this will throw an error. I have added a note for now.